node-scripts/spec/pipeline3/impl/dockerComposeGenericRegacy.spec.js

```diff
@@ -0,0 +1,192 @@
+"use strict";
+
+describe("Docker Compose Generic Deployment implementation", function() {
+  // deps
+  const path = require("path");
+  const config = require(path.resolve("src/utils/config"));
+  const cst = require(path.resolve("src/const"));
+  const dockerCompose = require(path.resolve(
+    "src/pipeline3/impl/dockerComposeGeneric"
+  ));
+
+  const scripts = require(path.resolve(
+    "src/" + config.getJobNameForPipeline3() + "/scripts"
+  ));
+
+  var instanceDef = {
+    uuid: "336af1ee-90a1-4d1b-baaf-db12c84deec0",
+    name: "cambodia1",
+    type: "dev",
+    group: "tlc",
+    deployment: {
+      hostDir: "/var/docker-volumes/",
+      type: "dockerComposeGenericMaven",
+      composePlugin: true,
+      dockerComposeFiles: ["docker-compose.yml", "docker-compose-2.yml"],
+      envFiles: ["env-file-1", "env-file-2"],
+      value: {
+        mavenProject: {
```

Comment:
Let's remove support for `mavenProject` as part of this new type.

Reply:
Please rename the file, if it's still needed, `dockerComposeGenericLegacy.spec.js`